src/api/helpers/get-populated-application/nominated-loss-payee/index.ts

```diff
@@ -0,0 +1,41 @@
+import { Context } from '.keystone/types'; // eslint-disable-line
+import decryptNominatedLossPayee from '../../decrypt-nominated-loss-payee';
+import { ApplicationNominatedLossPayee } from '../../../types';
+
+/**
+ * getNominatedLossPayee
+ * Get an application's nominated loss payee by ID
+ * @param {Context} KeystoneJS context API
+ * @param {String} lossPayeeId: Loss payee ID
+ * @returns {ApplicationPolicy} mapped policy
+ */
+const getNominatedLossPayee = async (
+  context: Context,
+  lossPayeeId: string,
+  decryptFinancialUk?: boolean,
+  decryptFinancialInternational?: boolean,
+) => {
+  try {
+    console.info(`Getting nominated loss payee ${lossPayeeId}`);
+
+    const nominatedLossPayee = await context.query.NominatedLossPayee.findOne({
+      where: { id: lossPayeeId },
+      query:
+        'id isAppointed isLocatedInUk isLocatedInternationally name financialUk { id accountNumber sortCode bankAddress vector { accountNumberVector sortCodeVector } } financialInternational { id iban bicSwiftCode bankAddress vector { bicSwiftCodeVector ibanVector } }',
```

Comment:
just an idea, we could keep these queries in a constant file where easier to dig up

Reply:
Completely agree, this would be great